# HG changeset patch
# User Rik <octave@nomad.inbox5.com>
# Date 1296076429 28800
#      Wed Jan 26 13:13:49 2011 -0800
# Node ID c69fd002a3fc0dd7742f6ef4365679757d3c2c4e
# Parent  ad636c1f353b6c2332bfbb565ac0c06b8d9f3054
Remove obsolete octave-bug.1 man page from distribution.

diff --git a/doc/ChangeLog b/doc/ChangeLog
--- a/doc/ChangeLog
+++ b/doc/ChangeLog
@@ -1,8 +1,13 @@
+2011-01-26  Rik  <octave@nomad.inbox5.com>
+
+	* interpreter/octave-bug.1: Delete obsolete man page
+	* interpreter/Makefile.am: Stop distributing octave-bug.1 man page.
+
 2011-01-26  Olaf Till <olaf.till@uni-jena.de>
 
 	* interpreter/package.txi: Document variables MKOCTFILE,
 	OCTAVE_CONFIG, and OCTAVE exported by pkg for configure and make.
 
 2011-01-25  Rik  <octave@nomad.inbox5.com>
 
 	* Makefile.am: Remove pass-through rules for AUTHORS, BUGS, and
diff --git a/doc/interpreter/Makefile.am b/doc/interpreter/Makefile.am
--- a/doc/interpreter/Makefile.am
+++ b/doc/interpreter/Makefile.am
@@ -31,17 +31,16 @@ TEXINPUTS := "$(PATH_SEPARATOR)$(srcdir)
 export TEXINPUTS
 
 ## Include custom texmf.cnf necessary to run @seealso macro 
 TEXMFCNF := "..$(PATH_SEPARATOR)$(srcdir)/..$(PATH_SEPARATOR)$(TEXMFCNF)$(PATH_SEPARATOR)"
 export TEXMFCNF
 
 dist_man1_MANS = \
   mkoctfile.1 \
-  octave-bug.1 \
   octave-config.1 \
   octave.1
 
 ## The following example files are listed for dependencies.
 ## They should not be distributed from this directory.
 EXAMPLE_FILES = \
   $(top_srcdir)/examples/@FIRfilter/display.m \
   $(top_srcdir)/examples/@FIRfilter/FIRfilter_aggregation.m \
diff --git a/doc/interpreter/octave-bug.1 b/doc/interpreter/octave-bug.1
deleted file mode 100644
--- a/doc/interpreter/octave-bug.1
+++ /dev/null
@@ -1,68 +0,0 @@
-.\" Copyright (C) 2000-2011 Dirk Eddelbuettel
-.\"
-.\" This file is part of Octave.
-.\"
-.\" Octave is free software; you can redistribute it and/or modify it
-.\" under the terms of the GNU General Public License as published by the
-.\" Free Software Foundation; either version 3 of the License, or (at
-.\" your option) any later version.
-.\"
-.\" Octave is distributed in the hope that it will be useful, but WITHOUT
-.\" ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
-.\" FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License
-.\" for more details.
-.\"
-.\" You should have received a copy of the GNU General Public License
-.\" along with Octave; see the file COPYING.  If not, see
-.\" <http://www.gnu.org/licenses/>.
-.\"
-.\" This page was contributed by Dirk Eddelbuettel <edd@debian.org>
-.\" 
-.TH octave-bug 1 "6 March 2000" GNU
-.SH NAME
-octave-bug \- report a bug in GNU Octave
-.SH SYNOPSIS
-.B octave-bug
-.RB [\| \-s
-.IR subject \|]
-.SH DESCRIPTION
-.B octave-bug
-is a shell script to help the user compose and mail bug reports
-concerning Octave in a standard format.  
-.B octave-bug
-is typically invoked by the Octave command
-.B bug_report
-which is intended to be called interactively from within Octave. This
-provides the best way to submit a bug report for Octave. It creates a
-template bug report file and starts an editor on that file.  The bug report
-will be sent to the bug-octave mailing list once the editing has been
-completed (this assumes of course that your system can use email). However,
-the user could also call \fBoctave-bug\fR outside of Octave.
-.PP
-Please read the `Bugs' chapter in the Octave manual to find out how to submit
-a bug report that will enable the Octave maintainers to fix the problem.  If
-you are unable to use the bug_report command, send your message to the
-bug-octave mailing list, bug@octave.org.
-.SH OPTIONS
-.TP
-.BI -s\  subject
-Specify a subject line for the bug report.  Spaces in the subject must
-be quoted.
-.SH ENVIRONMENT VARIABLES
-.B 
-octave-bug 
-uses the environment variables
-.BR USER, 
-.BR EDITOR, 
-and 
-.B PAGER 
-which can be used for customization.
-.SH VERSION
-This document was last revised for Octave version 2.0.16.
-.SH SEE ALSO
-.BR octave (1),
-.BR bashbug (1)     
-.SH AUTHOR
-.nf
-John W. Eaton
-<jwe@octave.org>
